veto/veto.cxx

```diff
@@ -198,23 +207,32 @@ double wx(double z){
 
 double wy(double z){
 
-  double wy1 = 2620 * mm;
-  double wy2 = 6120 * mm;
+  double wy1 = 3400. * mm;
+  double wy2 = 6000. * mm;
   double z1 = 0 * m;
   double z2 = 50 * m;
 
   return  (wy1 + (z - z1) * (wy2 - wy1) / (z2 - z1));
 }
 
 
-TGeoVolume* veto::GeoSideObj(TString xname, double dz,
-			     double a1, double b1,double a2, double b2,double dA, double dB,
-				Int_t color, TGeoMedium *material, Bool_t sens=kFALSE){
-
-  //a1- width in X, at the beginning
-  //b1- width in Y, at the beginning
-  //a2- width in X, at the end
-  //b2- width in Y, at the end
+TGeoVolume* veto::GeoSideObj(TString xname, 
+                              double dz,
+                              double a1,
+                              double b1,
+                              double a2,
+                              double b2,
+                              double dA,
+                              double dB,
+                              Int_t color,
+                              TGeoMedium *material,
+                              Bool_t sens=kFALSE)
+    {//TGeoVolume* LiScX = GeoSideObj(name, half space between ribs along z, width in X at the beginning,width in Y at the beginning,width in X at the end,width in Y at the end, change in width along X,change in width along y, kMagenta - 10, vetoMed, true);
+
+      //a1- width in X, at the beginning
+      //b1- width in Y, at the beginning
+      //a2- width in X, at the end
+      //b2- width in Y, at the end
```

Comment:
It would be best to reformat this comment in such a way that Doxygen can parse it for the automatic documentation, see e.g. here: https://en.wikipedia.org/wiki/Doxygen#Example_code